website/components/_app.tsx

```diff
@@ -73,6 +82,10 @@ export default function Nextra({ Component, pageProps }) {
           referrerPolicy="no-referrer-when-downgrade"
         />
       </noscript>
+      <CQCookieConsent
+        onAccept={() => setConsent(true)}
+        onDecline={() => setConsent(false)}
```

Comment:
When consent is withdrawn, should this also clear the existing cookies?

Reply:
Ok, as long as the scripts check the value of that cookie it should be fine. But the cookies are technically on our domain so I would expect we can delete/unset them

Reply:
It's done in https://github.com/cloudquery/cloudquery/pull/13494/files#diff-a4b784516284a9bb578fb84489f0480b3982436caeaace9b094c81297a65a48fR76